### PR TITLE
feat: Update JavaScript SDKs to v8.2.1 for stable release

### PR DESCRIPTION
**Pull request by timfish**
Updates to the stable release!

### CODE REVIEW

**Comment by Brooooooklyn** [comment]
@timfish let's release it, we are blocked by this: https://github.com/toeverything/AFFiNE/pull/6968

**Review by Lms24 — APPROVED** [review]
Sorry for the late review - looks good, thanks Tim!

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Unreleased
 
-## 5.0.0-beta.0
+## 5.0.0
```
Comment (Lms24) [inline CHANGELOG.md:5]: Given the changelog entry is what many people will read first, can we just add the say 3 most important changes to it and link to respective docs? I think the specific imports for renderer and main process might be a candidate. wdyt?

**Comment by timfish** [comment]
Follow this issue to be notified of the release 
https://github.com/getsentry/publish/issues/3875

**Pull request merged**